src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -1663,7 +1630,8 @@ k
             cc.fillStyle = 'transparent';
             this.canvas_draw_poly(cc, area.coordinates);
 
-            cc.fillStyle = COLORS.RESTRICTED_AIRSPACE_FILL;
+            // FIXME: Is the restricted airspace EVER filled???
+            cc.fillStyle = this.theme.RESTRICTED_AIRSPACE_FILL;
```

Comment:
It's not "filled", but apparently that's what `context.fillText()` uses for coloring. Restored, fixed bad reference. All working better now. :+1: